hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/snapshot/TestOMSnapshotCreateRequest.java

```diff
@@ -238,6 +247,50 @@ public void testValidateAndUpdateCache() throws Exception {
         omResponse.getStatus());
   }
 
+  @Test
```

Comment:
If you temporarily remove the logic in `OMSnapshotCreateResponse` you can add `@Ignore` to this UT so that it is not executed. Add a TODO here as well.

Reply:
I'll keep the logic here for now and move it once locking patch is merged.